cloudferrylib/os/identity/keystone.py

```diff
@@ -213,6 +214,8 @@ def deploy(self, info):
         self._upload_user_tenant_roles(roles, users, tenants)
         LOG.info("Done")
 
+    @retrying.retry(expected_exceptions=[ks_exceptions.Unauthorized],
+                    reraise_original_exception=True, max_attempts=5)
```

Comment:
As SVilgelm is pointing out, I had tried CONF variables but it could not use there. Also to use this I need to ass one global variable which is again against the standard. That's why I had put a hard coded value. 
So in next, I will consider retry in run function itself.